cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -826,4 +887,41 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         declStmt.addDeclaration(decl)
         return declStmt
     }
+
+    /** Creates an implicit [AssignExpression]. */
+    private fun createImplicitAssignStatement(lhs: Expression, rhs: Expression): AssignExpression {
+        return newAssignExpression(operatorCode = "=", lhs = listOf(lhs), rhs = listOf(rhs))
+            .implicit()
+    }
+
+    /** Creates an implicit [MemberCallExpression] (e.g., `__enter__`, `__exit__`). */
+    private fun createImplicitMemberCallExpression(
+        name: String,
+        base: Expression,
+        item: Python.AST.withitem
+    ): MemberCallExpression {
+        return newMemberCallExpression(
+                callee = newMemberExpression(name = name, base = base),
+                rawNode = item
+            )
+            .implicit()
+    }
+
+    /** Creates a [TryStatement]. */
+    private fun createTryStatementWithBody(
```

Comment:
I would avoid these extra functions. We are currently looking into how to give the node creation a more "fluid" feeling and once that is done this will probably easier but in the mean-time I would avoid these extra functions which are basically just one-liners